index.html

```diff
@@ -813,13 +788,13 @@ <h3>Extensions to the <code>Performance</code> Interface</h3>
               attribute EventHandler onresourcetimingbufferfull;
 };</pre>
 <p>The <dfn>Performance</dfn> interface is defined in
-[[!HR-TIME-2]].</p>
+[[HR-TIME-2]].</p>
 <p>The method <dfn>clearResourceTimings</dfn> runs the following
 steps:</p>
 <ol>
 <li>remove all <a>PerformanceResourceTiming</a> objects in the
 <a data-cite=
-'!PERFORMANCE-TIMELINE-2/#dfn-performance-entry-buffer'>performance
+'!PERFORMANCE-TIMELINE-2#dfn-performance-entry-buffer'>performance
```

Comment:
So here we need the "!" prefix? If so why? WHY???????

Reply:
Also, can you point me to relevant docs that'd allow me to educate myself on that front?

Reply:
Oh, you missed the memo 📝 https://lists.w3.org/Archives/Public/spec-prod/2018OctDec/0000.html

We now use “artificial intelligence” (i.e., “if statements”) for citations, so ReSpec knows if a reference is normative or informative:)

Reply:
So, we don’t actually need “!” anywhere, my mistake when I did find/replace.

Reply:
I'm glad our robot overlords are being kind with us. Can you remove the '!' then?

Reply:
Yeah, remove it. Sorry, already left the office, so on my phone... otherwise would do it now.

Reply:
will do!